support-frontend/assets/pages/[countryGroupId]/validation.ts

```diff
@@ -19,3 +19,4 @@ export function preventDefaultValidityMessage(
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape
  */
 export const doesNotContainEmojiPattern = '^[^\\p{Emoji_Presentation}]+$';
+export const numericOnlyPattern = '^[0-9]*$';
```

Comment:
Should this not be validated by having [`<input type="number" />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number)?

Reply:
Ah yes, much better than using un-required regex. Changed.